### PR TITLE
Make form button reveal message when clicked 

### PR DESCRIPTION
**Pull request by RCAVelez**

### CODE REVIEW

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css [hunk 1]
```diff
@@ -24,9 +70,11 @@ img {
 
 .header-container {
   align-items: center;
+  background-color: grey;
   display: flex;
   height: 100px;
   justify-content: space-evenly;
+  margin-bottom: 10px;
   margin-right: auto;
   margin-left: auto;
```
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css]: at this point you might want to combine to `margin: 0 auto 10px auto;` (not sure what you want for margin-top)

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,68 +1,107 @@
...
+      <a class="footer-item" href="#work" onclick="animateScrollTop('work-container')">My Work</a>
+      <a class="footer-item" href="#contact">Contact</a>
+    </div>
+    <div class="column2">
+      <p>Contact Me!</p>
+      <p class="form-message">We'll be in touch!</p>
+      <input type="text" name="" placeholder="First Name">
+      <input type="text" name="" placeholder="Last Name">
+      <input type="text" name="" placeholder="Email">
+      <button type="button" name="button" onclick="message()">Send</button>
```
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:102]: what's this `message` function?

**Review by genedwards — CHANGES_REQUESTED** [review]

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css (hunk 1 above)
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css]: done

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html (hunk 2 above)
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:102]: added to script.js

Comment (genedwards) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:102]: I'm going back on this from a previous comment, I think it's worth removing the `onclick` attribute and hooking up $(...).click(() => {}) handler via JS.
Reasons being mixing of template and logic lacks separation of concerns, it's harder to find what function actually calls message(), and it's more difficult to test.

I'm ok doing it in a separate PR though. Please open a separate PR to fix this once this PR is merged.

**Review by genedwards — APPROVED** [review]
Please see comment on separate PR, otherwise LGTM.
Also as a separate effort, we can work on improving the loaded images to get a correct aspect ratios.

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html (hunk 2 above)
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:102]: will do!

**Pull request merged**